tests/storage/linstor/test_linstor_sr_intrapool_migration.py

```diff
@@ -9,7 +9,6 @@
 # - A VM to import to the LINSTOR SR
 # And:
 # - access to XCP-ng RPM repository from hostA1
-# - a repo with the LINSTOR RPMs must be given using the command line param `--additional-repos`
 
 @pytest.mark.small_vm # run with a small VM to test the features
 @pytest.mark.big_vm # and ideally on a big VM to test it scales
```

Comment:
Isn't this test file missing a live *shared* migration test (see NFS tests) ?

Reply:
Nice catch, I checked the tests of few drivers to repair the LINSTOR part, but not NFS.